docs-team/2016-09/september-22.md

```diff
@@ -0,0 +1,143 @@
+- DB: https://github.com/ember-learn/ember-help-wanted
+- DB: Tried firebase to host,  didn't work out - can't to querying very well, especially with permissions.  Tried couchdb, not a good fit.  Still looking for a backend technology.  Possibly Denali?  Tom Dale has been looking at it. Dave wasmer(?) is the contact on that.  Dave is willing to pay for backend hosting with the purpose of stress testing the technology.
+
+### Website
+
+- RM: Growing not to like the web site for various reasons.  Will ask the team for ideas on what do with it.  Intends to send Leah a proposal.
+- LS: Kris Seldon introduced me to someone wanting to redesign, but fell through.
+- RM: Get me your thoughts on what we need to do the web site
+
+### Blog post
+
+- RM: have not got to it but will now come to the top of my list
+
+### Learn Page
+
+- RM: To have a talk with Yehuda about FAQ and some other learn page related stuff around onboarding and repeat experience.  Roadmap/status board may go to builds page, but we need to work on it.
+
+### "Fancifying" Guides
+
+- RM: Take inspiration from O'Reily to have footnotes and callouts, nicer way to provide links.  
+- RM: Interactivity to switch between classic code paths and module unification
+- RM: Runnable code samples (twiddle)
+
+### Future Proofing Guides
+
+- A lot of people still want to use classic actions, and we don't do a good job explaining that there are two types of actions.  The currently have to go to older guides versions to read about them.  Matters because of module unification and glimmer components.  Need to be able to introduce concepts and differentiate paths.
+- DB: Concerned about the amount of work.  I see that the priority is clarity of guides over new features, such as classic vs closure, etc.  I think the primary thing to work through is to work through these different paths.
+- TJ: I think we need to do a better job around explaining component data flow (DDAU).  Our guides are pretty into fine detail, but don't introduce higher level concepts.  There are a few blog posts that do a decent job at this.
+- DB: could you compile a list of these, so maybe we could ask some of these authors if they want to contribute concepts.
+- RM: This is an overall comment for guides.  I think we can do a better job all around when explaining high level conceptual stuff.  Maybe separate our sections.
+
+### More Learn Page
+
+- DB: would you (LS) agree that the learn page is blocked on styling?
+- RM: It could look better, but I think we are more blocked on advanced topics
+- DB: we should save advanced topics for later and expose it then.
+- RM: Lets resolve the menu bar discussion then.
+- LS: I think its intuitive that "Learn" would lead to guides and api, but I know that not everyone would agree.
+- RM: its what angular does
+- LS: could we just replace guides with Learn and leave api
+- DB: I would like to replace both guides and api with learn, but I know there would be pushback
```

Comment:
API is my most-pointed-to link when sitting at other peoples' desks, but I directly hit the URL at this point.